### PR TITLE
ardour: 6.2 -> 6.3

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/ardour/default.nix [hunk 1]
```diff
@@ -12,17 +12,17 @@
 , fftw
 , fftwSinglePrec
 , flac
-, fluidsynth
+# , fluidsynth
```
Comment (roberth) [inline pkgs/applications/audio/ardour/default.nix:15]: Please add a comment explaining when these should be added back in or remove entirely.

**Comment by eadwu** [comment]
Can't really tell if this is working, no GUI is popping up and no errors are showing.
```
Ardour: [INFO]: Loading plugin meta data file /nix/store/ccw9cqlmxi7lbpihrs0f8d69anpqhnsf-ardour-6.3/share/ardour6/plugin_metadata/plugin_tags
Ardour: [INFO]: Loading default ui configuration file /nix/store/ccw9cqlmxi7lbpihrs0f8d69anpqhnsf-ardour-6.3/etc/ardour6/default_ui_config
Ardour: [INFO]: Loading 449 MIDI patches from /nix/store/ccw9cqlmxi7lbpihrs0f8d69anpqhnsf-ardour-6.3/share/ardour6/patchfiles
Ardour: [INFO]: Loading color file /nix/store/ccw9cqlmxi7lbpihrs0f8d69anpqhnsf-ardour-6.3/share/ardour6/themes/dark-ardour.colors
```

File: pkgs/applications/audio/ardour/default.nix (hunk 1 above)
Comment (magnetophon) [reply pkgs/applications/audio/ardour/default.nix]: I removed them entirely, as they are unneeded.

**Comment by magnetophon** [comment]
@eadwu That is really odd.  For one it just works here, but also: in the past, when ardour broke for some reason, it has always given an error.
Could you try again please?

**Comment by eadwu** [comment]
Ah found out the reason, https://discourse.ardour.org/t/ardour-doesnt-start-but-doesnt-crash-just-hangs-there-stuck/89118. Took a while to find since I didn't have an explicit error, anyway working fine on my end after the fix `GTK2_RC_FILES=/nonexistent ardour6`.

Anyway working fine, though to silence the gtk errors one might wrap `GTK_PATH` with the `/lib/gtk-2.0` paths in `gtk2` and `gnome3.gnome-themes-extra`.

**Review by veprbl — APPROVED** [review]
LGTM

**Comment by roberth** [comment]
Thanks @magnetophon @eadwu and @veprbl!

**Pull request merged**